x-pack/plugins/cloud_security_posture/public/pages/findings/use_findings.ts

```diff
@@ -88,8 +88,8 @@ export const useFindings = ({ index, query, sort, from, size }: UseFindingsProps
   return useQuery(
     ['csp_findings', { from, size, query, sort }],
     () =>
-      data.search
-        .search({
+      lastValueFrom<IEsSearchResponse<CspFinding>>(
```

Comment:
this bit with `lastValueFrom` was not added by us and i had conflicts in this file after rebasing, which led to minor type changes.